main.go

```diff
@@ -388,6 +404,9 @@ func loadConfig(filename string, queryUserData bool) (*Config, error) {
 		TasksDir:                       defaultTasksDir(),
 		WorkerGroup:                    "test-worker-group",
 		WorkerTypeMetadata:             map[string]interface{}{},
+		Project:                        "",
```

Comment:
This is to make sure whoever is using g-w remembers to set up a proper project name.